cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -247,13 +247,8 @@ func CertOrKeyExist(pkiPath, name string) bool {
 
 	_, certErr := os.Stat(certificatePath)
 	_, keyErr := os.Stat(privateKeyPath)
-	if os.IsNotExist(certErr) && os.IsNotExist(keyErr) {
-		// The cert and the key do not exist
-		return false
-	}
 
-	// Both files exist or one of them
-	return true
+	return !(os.IsNotExist(certErr) && os.IsNotExist(keyErr))
```

Comment:
i think the original is better and provides slightly more clarity with the comments.
are you getting this as an error from a linter?

Reply:
The next function of this function is written like this; I think this is more concise, and the logic itself is not complicated

Reply:
so i change it by the way

Reply:
> The next function of this function is written like this; I think this is more concise, and the logic itself is not complicated

ok, that is true.